ui/page/root/home_page.go

```diff
@@ -170,6 +174,193 @@ func (hp *HomePage) OnNavigatedTo() {
 	hp.isBalanceHidden = hp.AssetsManager.IsTotalBalanceVisible()
 }
 
+type dexWalletUnlockInfo struct {
+	passEditor cryptomaterial.Editor
+	wallet     sharedW.Asset
+}
+
+// initDEX initializes a new dex client if dex is not ready.
+func (hp *HomePage) initDEX() {
+	if hp.AssetsManager.DexcReady() {
+		return // do nothing
```

Comment:
Should this even be possible?

Even if it is, we do still need to check if we need to unlock wallets. And also start the listener for the bond expiry.